server/scripts/get-email-list.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-console */
+const { dbClient } = require('../db');
+require('dotenv').config({ path: '../.env' });
+(async function exec() {
+  await dbClient.connect();
+  const res = await dbClient.db.query(`
+    SELECT p.body ->> 'emailAddress' AS email
+    FROM participants p 
+    WHERE 
+        p.body ->> 'emailAddress' NOT IN (
```

Comment:
This seems like a kinda convoluted way of only grabbing those participants without `hired` statuses. Would a standard join do the trick? BRB with a suggestion.

Reply:
I initially went with a standard join, but it doesn't account for the case where a user has multiple `participants_status`, with one of them being 'hired'. It will still join with the non-hired `participants_status`. 

Putting the exclusion in the where clause isn't the most efficient, but it's the simplest way of excluding an email.